woocommerce/woocommerce-gutenberg-products-block
# Add .prettierignore configuration

This PR adds the `.prettierignore`. In this way, Prettier excludes `.scss` files.


Fixes #5064 




### Manual Testing

How to test the changes in this Pull Request:

Check out this branch
Check that your editor has Prettier configured correctly and it read the configuration of this project

1. Open a `.scss` file.
2. Check that the editor, by adding some space, does not format the code according to Prettier's rules.
